src/sidebar/annotation-service.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
I realise now that the name is perhaps a bit confusing because it suggests it might be an _Angular service_. Perhaps rename it to something `serviceConfig` / `service-config.js`?